src/metrics/database_definitions.go

```diff
@@ -45,6 +45,8 @@ func (q *QueryDefinition) insertDatabaseNames(databases collection.DatabaseList)
 var databaseDefinitionUnder91 = &QueryDefinition{
 	query: `SELECT -- UNDER91
 		D.datname AS database,
+		/* This query retrieves max_connections from Postgres configuration settings to facilitate calculation of connections percentage (db.connections/db.maxconnections*100). */
```

Comment:
I would suggest to put this as a go comment above the `var databaseDefinitionUnder91`, rather than in the SQL (which should work fine, but might be slightly confusing).

e.g.
```go
// databaseDefinitionUnder91 is the query used to fetch metrics from Postgres version 9.1 and below.
// As a special case, max_connections is obtained from the pg_settings table rather than from pg_stat_database
var databaseDefinitionUnder91
```

Reply:
ok. I will make that change 👍